content/games/factions/routines.md

```diff
@@ -0,0 +1,143 @@
+     zone.
+   - If the result is _odd_, deploy the group furthest left in the line to the furthest deployment
+     zone.
+1. For each remaining Group, roll 1d6:
+   - Select the Group from the left/right of the line if the result is odd/even respectively.
+   - If the result is _lower_ than the previous roll, deploy in an adjacent quadrant.
+   - If the result is _equal_ to the previous roll, deploy in the same quadrant.
+   - If the result is _higher_ than the previous roll, deploy in the opposite quadrant.
+1. Depending on the Company's Personality, these deployments may need to be adjusted, either
+   swapping Groups or moving them within their deployment quadrant:
+   - Arcane Companies must deploy Groups with the Casting or Summoning order within 6" of a friendly
+     group if possible.
+   - Bold Companies must deploy their Groups as near an enemy or objective as possible.
+   - Cautious Companies must deploy their Groups with Shooting, Casting, or Summoning orders within
+     6" of a friendly group and as far from enemy Groups as possible.
+   - Ferocious Companies must deploy their Groups within range to Attack or Shoot if possible.
+   - Guarded Companies must deploy their groups within 6" of a friendly Group. If any Groups have
+     the Defensive trait, they must be evenly dispersed amongst Group clusters.
+
+### Ambitions
+
+Routine Companies may have [Ambitions]({{% ref "/games/factions/rules#ambitions" %}}). For each
+Ambition listed for the Personality, roll a d6: on a 4+, they have that Ambition. Routine Company
+Ambitions always count as revealed.
+
+- **Arcane:** Give Better Than We Get, Draw First Blood,They Will Know My Name
+- **Bold:** They Will Reel Before Us, Give Better Than We Get, Draw First Blood, They Will KNow My
+  Name
+- **Cautious:** We Will Not Break, Give Better Than We Get, Rain Hell Upon Them
+- **Ferocious:** They Will Reel Before Us, We Will Snap Their Spine, Break Even Their Anvil, Draw
+  First Blood, We Will All Shed Blood
+- **Guarded:** We Will Not Break, Win Without Moving, Do Not Waver, Give Better Than We Get, Rain
+  Hell Upon Them
+- **Incompetent:** None.
+
+## Turns
+
+Routine Companies start their turn as normal, rallying Shaken Groups then testing for Reckless
+Groups.
+
+When activating individual Groups, unless specified, they follow this process:
+
+1. Must activate the Captain's Group first, then the closest Group to the Captain's Group which has
+   not yet activated.
+1. Roll 1d6 to determine the activation: Move (1--2), Attack (3--4), Shoot (5--6). If the group
+   cannot perform the specified action, default to the next lower-order action until they can.
+   - If Moving, roll 1d6 and: Move away from nearest enemy (1--2), Move towards nearest enemy
+     (3--4), Move towards nearest objective (5--6).
+1. Resolve each activation as normal.
+
+### Arcane Group Activations
+
+Arcane Companies must activate any Groups with the Casting or Summoner order before others, from
```

Comment:
I had this as `order` here to account for the Apprentice/Caster traits, both of which have the `Cast` order but I see I fucked up and said `Casting or Summoner` instead of `Cast or Summon`. I'm fine going either to the orders or the traits.

Reply:
Leaning on orders is better, actually. Good call.